tests/test_crossref.py

```diff
@@ -49,6 +49,18 @@ def test_transform_dict_with_metadata_mapping_full_metadata(
     assert metadata == dspace_metadata
 
 
+def test_is_valid_dspace_metadata_failure():
```

Comment:
It might be a good idea to add more examples of invalid metadata (other than empty key/value)?